src/lib/work_partitioner/pending_zkapp_command.ml

```diff
@@ -17,6 +17,15 @@ type t =
   ; mutable merge_remaining : int
   }
 
+let create ~job ~unscheduled_segments ~pending_mergeable_proofs ~merge_remaining
+    =
+  { job
+  ; unscheduled_segments
+  ; pending_mergeable_proofs
+  ; elapsed = Time.Span.zero
+  ; merge_remaining
```

Comment:
I wonder what're semantics behind `merge_remaining` and could it be defined from other fields (`job`, `unscheduled_segments`, `pending_mergeable_proofs`)?